portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,124 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Fiana Vicker's Professional Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body = style = "border-top: lightslategray;
+                   background-color: grey; background-image: url('images/wooddark.jpeg');
+                   
+                   " >
+
+    <div id="banner">
+        <h2 style="top: 0px; background-color: palevioletred; color: lightgray;">
+                Personal Projects        |        Gallery       |          Professional Projects      
```

Comment:
Browsers typically treat any amount of consecutive whitespace as a single space. If you want to space these out, I would try CSS flexbox to lay them out horizontally and space them out.